python/src/main/db.py

```diff
@@ -64,9 +63,17 @@ def get_person(self, email):
 
     def delete_person(self, email):
         self.cursor.execute("DELETE FROM Person WHERE email = '%s'" % email)
+        self.cursor.execute("DELETE FROM Item WHERE person_email = '%s" % email)
+        self.conn.commit()
+
```

Comment:
Whats the difference between the delete from item lines in delete_person and delete_all_persons_items and delete_item?
Can we reuse the functionality?

Reply:
When you say reuse functionality, does that mean for example within delete_person, calling the delete_all_persons_items instead of having the extra line of code OR trying to condense the 2 (or 3) functions into one more versatile function?